src/tsoding_bot.app.src

```diff
@@ -0,0 +1,16 @@
+{application, tsoding_bot,
+ [{description, "An OTP application"},
+  {vsn, "0.1.0"},
+  {registered, []},
+  {mod, { tsoding_bot_app, []}},
+  {applications,
+   [kernel,
+    stdlib
+   ]},
+  {env,[]},
+  {modules, []},
+
+  {maintainers, []},
+  {licenses, []},
```

Comment:
You should probably fill `maintainers` and `licenses` fields to avoid confusion.